packages/EasyDecision/src/Bridge/Symfony/DependencyInjection/EasyDecisionExtension.php

```diff
@@ -43,5 +46,12 @@ public function load(array $configs, ContainerBuilder $container): void
         $container
             ->autowire(MappingProviderInterface::class, ConfigMappingProvider::class)
             ->setArgument('$decisionsConfig', $config['type_mapping'] ?? []);
+
+        // Register middleware if messenger present
+        if (\class_exists(MessengerPass::class)) {
+            $middlewareDefinition = new Definition(ResetDecisionMiddleware::class);
+
+            $container->setDefinition(ResetDecisionMiddleware::class, $middlewareDefinition);
```

Comment:
let's use a php config file to register the middleware so we can use autowire feature on it. Have a look at other packages to see how they use the `PhpFileLoader`.